python/ray/remote_function.py

```diff
@@ -100,6 +101,9 @@ def __init__(self, language, function, function_descriptor, num_cpus,
                            if max_calls is None else max_calls)
         self._max_retries = (DEFAULT_REMOTE_FUNCTION_NUM_TASK_RETRIES
                              if max_retries is None else max_retries)
+        self._retry_exceptions = (DEFAULT_REMOTE_FUNCTION_RETRY_EXCEPTIONS
```

Comment:
Can you update the doc string for the class, and the one for @ray.remote in `worker.py`?